lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -285,14 +289,14 @@ Try {
             If(-not $_ansible_check_mode) {
                 If(-not $workgroup_match) {
                     Write-DebugLog ("setting workgroup to {0}" -f $workgroup_name)
-                    $join_wg_result = Join-Workgroup -workgroup_name $workgroup_name -domain_admin_user $domain_admin_user -domain_admin_password $domain_admin_password
+                    Join-Workgroup -workgroup_name $workgroup_name -domain_admin_user $domain_admin_user -domain_admin_password $domain_admin_password
```

Comment:
`Join-Workgroup` can call `Set-Workgroup` which returns something, we need to redirect to `$null`.

Reply:
changing it breaks the CI - you want to keep it or should I look into changing the tests?